.rake_tasks~

```diff
@@ -0,0 +1,86 @@
+acceptance
```

Comment:
Looks like this a temporary file. Could you remove it?

Reply:
Ok, I will fix it.